target/linux/pistachio/dts/pistachio/pistachio_marduk_common.dtsi

```diff
@@ -17,6 +17,7 @@
 		serial0 = &uart0;
 		serial1 = &uart1;
 		ethernet0 = &enet;
+		wifi0 = &wifi;
```

Comment:
Surely two aliases pointing directly to each address would be more generic than pointing to the parent node?